forge/ee/db/views/PipelineStage.js

```diff
@@ -1,11 +1,20 @@
 module.exports = function (app) {
+    app.addSchema({
+        $id: 'DevicesSummaryList'
+
+    })
+
     app.addSchema({
         $id: 'PipelineStage',
         type: 'object',
         properties: {
             id: { type: 'string' },
             name: { type: 'string' },
-            instances: { type: 'array', items: { ref: 'InstanceSummaryList' } },
+            instances: { $ref: 'InstanceSummaryList' },
+            devices: {
+                type: 'array',
+                $ref: 'InstanceSummary'
```

Comment:
Not sure this is meant to be `InstanceSummary` - looks like a `DevicesSummaryList` that is defined above but otherwise unused.